plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -388,7 +388,7 @@ public JdbcProcedureHandle getProcedureHandle(ConnectorSession session, Procedur
     public void commitCreateTable(ConnectorSession session, JdbcOutputTableHandle handle, Set<Long> pageSinkIds)
     {
         delegate.commitCreateTable(session, handle, pageSinkIds);
-        invalidateTableCaches(new SchemaTableName(handle.getSchemaName(), handle.getTableName()));
+        invalidateTableCaches(new SchemaTableName(handle.getRemoteTableName().getSchemaName().get(), handle.getRemoteTableName().getTableName()));
```

Comment:
Should not you use `orElse(null)` here? Like it was before? Or `orElseThrow()` to mention that it can fail here.

Also you may want to add `SchemaTableName RemoteTable::geSchemaTableName()`. I see it is called multiple times.

Reply:
The `SchemaTableName` require the schema nonnull